README.md

```diff
@@ -1 +1,37 @@
-# spec
+# OpenEvents
+
+Events are everywhere.  However, event publishers tend to describe events differently.
+
+The lack of a common way of describing events means developers must constantly re-learn how to receive events.  This also limits the potential for libraries, tooling and infrastructure to aide the delivery of event data across environments, like SDKs, event routers or tracing systems.  The portability and productivity we can achieve from event data is hindered overall.
+
+Enter OpenEvents, a specification for describing event data in a common way.  OpenEvents seeks to ease event declaration and delivery across services, platforms and beyond.
+
+OpenEvents is a new effort and it's still under active development.  However, its working group has received a surprising amount of industry interest, ranging from major cloud providers to popular SaaS companies.  Our end goal is to offer this specification to the [Cloud Native Computing Foundation](https://www.cncf.io/).
+
+## Roadmap
+
+*January*
+
+* Establish governance, contributing guidelines and initial stakeholders.
```

Comment:
Might be good to include things like: gather/agree to usecases and design goals for v1

Reply:
This one a good point. On one hand, it is useful to list all the use cases where event-driven architecture applies.  On the other hand, it will keep the spec focussed if we define which use cases we are targetting and which we are explicitly leaving out at this stage.

Reply:
I've added "agree upon design goals" to the Road map.  Unsure about whether to limit the use-cases.  We should discuss.

Reply:
You are right. It may be too early to have a conversation about limiting the use cases. 👍  for "agree upon design goals"